src/androidTest/java/at/bitfire/ical4android/ICalendarTest.kt

```diff
@@ -70,6 +70,15 @@ class ICalendarTest {
 		assertEquals("#123456", calendar.getProperty<Property>(ICalendar.CALENDAR_COLOR).value)
 	}
 
+	@Test
+	fun testFromReader_invalidProperty() {
+		javaClass.classLoader!!.getResourceAsStream("events/invalid-geo.ics").use { stream ->
+			ICalendar.fromReader(
+				stream.reader()
+			)
+		}
+	}
```

Comment:
I think this test is at the right place and does what it should. However:

1. I'd use a StringReader so that we don't have an external file dependency (see `testFromReader_calendarProperties`).
2. Reducing the iCalendar to a minimum number of properties so that it's still valid, but shows the test case would allows to test only what actually should be tested in this test.

Reply:
I've compressed it into a187acef7941c4f22ac801b8faa13d46ba8a1a5a